src/components/form/TextField/TextField.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { AppTheme } from '~/models/AppTheme ';
+import renderWithRedux from '~/utils/testing/renderWithRedux';
+
+import TextField from './TextField';
+
+describe('<TextField />', () => {
+  it('should render error message if field touched and has invalid value ', async () => {
+    const errorText = 'Required';
+    const { getByTestId } = renderWithRedux(
+      <TextField error={errorText} touched />
+    );
+
+    const expectedErrorText = getByTestId('errorMessage').children[0];
+
+    expect(expectedErrorText).toBe(errorText);
```

Comment:
On naming convention, I think something like this would be better:
```js
// or `actualErrorText` or just `errorText`
const renderedErrorText = getByTestId('errorMessage').children[0];
expect(renderedErrorText).toBe(expectedErrorText);
```
And similarly in further tests.